src/lib/component/Icon.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import Icon from 'svelte-icons-pack/Icon.svelte';
+
+  export let src: string | { a: { viewBox: string }; c: string } | undefined;
```

Comment:
just curious about the second type. How does it work?

Reply:
The second type is the type corresponding to some of the icons in the icon pack package we are using are organized, I believe its structure comes from the way the svg is formatted.